<launch>
  <arg name="default_frame_id" default="odom_on_ground" />

  <param name="torus_udiv" value="40" />
  <param name="torus_vdiv" value="40" />

  <!-- transformable marker server -->
  <node pkg="jsk_interactive_marker" type="transformable_server_sample" name="transformable_interactive_server"
        output="screen">
   <param name="display_interactive_manipulator" value="true"/>
  </node>
  <node pkg="drc_task_common" type="drill_button_publisher.py" name="drill_button_publisher"/>
  <node pkg="drc_task_common" type="move_marker.py" name="move_marker"/> 
</launch>
